tests/test_custom_checkers/test_type_annotation_checker.py

```diff
@@ -12,12 +12,12 @@ def test_type_is_assigned_for_function(self):
         """Ensure that checker catches when type is assigned instead of annotated
         in function parameters."""
         src = """
-        from typing import List
+        from typing import list
```

Comment:
Delete this line (and the blank lines below it)

Reply:
This change leads to a failing test in Python 3.8. As such, it has been reverted.